src/components/Notebook.jsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React from 'react';
```

Comment:
I think airbnb rule gave me single quotes, is this cool?

Reply:
I'm cool